xenit-alfresco/templates/acs/network-policy.yml

```
@@ -230,4 +230,46 @@ spec:
         - protocol: TCP
           port: 8443
 ---
+#Do we only enable these rules on replicas > 1?
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: acs-to-acs
+spec:
+  podSelector:
+    matchLabels:
+      app: acs
+  policyTypes:
+    - Egress
+  egress:
+    - to:
+        - podSelector:
+            matchLabels:
+              app: acs
+      ports:
+        - protocol: TCP
+          port: 5701
```

You can put both egress & ingress policies in the same spec.
This is especially useful when we're talking about clustering scenarios where the application is talking to itself (technically, it's replicas)

I have not checked the indentation of this block; as that's a bit hard in the github comment UI, so doublecheck that indentation is correct before just applying this suggestion

```suggestion
  policyTypes:
    - Ingress
    - Egress
  egress:
    - to:
        - podSelector:
            matchLabels:
              app: acs
      ports:
        - protocol: TCP
          port: 5701
  ingress:
      - from:
          - podSelector:
              matchLabels:
                app: acs
        ports:
          - protocol: TCP
            port: 5701
```

---

Thanks for the feedback, I'll try it out first on liantis to confirm;

(unrelated)
https://docs.hazelcast.com/hazelcast/5.5/phone-homes 
for disabling.